crates/rpc-types/src/eth/transaction/request.rs

```diff
@@ -211,12 +212,14 @@ impl TransactionRequest {
     /// If required fields are missing. Use `complete_legacy` to check if the
     /// request can be built.
     fn build_legacy(self) -> TxLegacy {
+        let checked_to = self.to.expect("the `to` field should have value.");
```

Comment:
let's make these match the others `to: self.to.expect("checked in complete_legacy"),`